templates/tracker/donate.html

```diff
@@ -373,6 +373,17 @@
 		{% endif %}
 		<br />
 
+<!-- Steam Stuff -->
+{% if request.session.uid %}
+  <p>Connected to steam!</p>
+  <form action="{% url 'social:disconnect' 'steam' %}?next={{ request.get_full_path }}" method="POST">
+      {% csrf_token %}
```

Comment:
Used to make Django not complain about a potential XSS attack
https://docs.djangoproject.com/en/1.10/ref/csrf/#how-to-use-it